src/pages/waiting-room-to-car/__tests__/waiting-room-to-car.spec.ts

```diff
@@ -112,84 +116,25 @@ describe('WaitingRoomToCarPage', () => {
     it('should get tell me question from the question provider', () => {
       expect(component.tellMeQuestions.length).toBe(2);
     });
-    describe('selecting a tell me question', () => {
-      it('should dispatch an action when the tell me question change handler is called', () => {
-        const question: TellMeQuestion = {
-          code: 'T1',
-          description: 'desc',
-          shortName: 'name',
-        };
-        component.tellMeQuestionChanged(question);
-        expect(store$.dispatch).toHaveBeenCalledWith(new TellMeQuestionSelected(question));
-      });
-    });
+
+    // TODO move to a test for the new component
```

Comment:
Commented test. I don't think this test needs to be moved/changed, it relates to the action being dispatched by the container component.

Reply:
uncommented.